src/modals/reveal-private-key-modal/reveal-private-key-view/reveal-private-key-view.tsx

```diff
@@ -20,6 +21,20 @@ export const RevealPrivateKeyView: FC<Props> = ({ publicKeyHash }) => {
     setSecretKey(undefined);
   }, [publicKeyHash]);
 
+  useEffect(() => {
+    if (isDefined(secretKey)) {
+      const timer = setTimeout(() => {
+        setSecretKey(undefined);
+      }, 3 * 60_000);
+
+      return () => {
+        clearTimeout(timer);
+      };
+    }
+
+    return undefined;
+  }, [secretKey, setSecretKey]);
+
```

Comment:
why did you create this `useEffect` if there is already exists a timer that hides the secret key?
```
activeTimer.current = setTimeout(() => setSecretKey(undefined), OVERLAY_SHOW_TIMEOUT);
```